docs/index.md

```diff
@@ -133,6 +133,22 @@ attendances_features_fg.show(1)
 
 You can find more examples on how to use the library in our [hops-examples](https://github.com/logicalclocks/hops-examples) repository.
 
+## Usage
+
+Usage data is collected for improving quality of the library. It is turned on by default if the backend 
+is "c.app.hopsworks.ai". To turn it off, use one of the following way:
+```python
+# use environment variable
+import os
+os.environ["ENABLE_HOPSWORKS_USAGE"] = "false"
+
+# use `disable_usage_logging`
+import hsfs
+hsfs.disable_usage_logging()
+```
```

Comment:
it make it easier to understand that they are 2 different options (but either way is fine)
```suggestion
```python
# use environment variable
import os
os.environ["ENABLE_HOPSWORKS_USAGE"] = "false"
```
```python
# use `disable_usage_logging`
import hsfs
hsfs.disable_usage_logging()
```
```